### DISCUSSION TITLE
after I use "go test xxx 2>&1 | go-junit-report > result.xml", then the console doesn't print any more information

### DISCUSSION THREAD

**Discussion by StevenIIV** [jstemmer/go-junit-report#153] [Q&A]

**Comment by jstemmer** [comment]
What did you expect to see, the output from `go test`? What version of go-junit-report are you running?

**Comment by StevenIIV** [comment]
yes, the output from go test, version is V 1.0.0

**Comment by StevenIIV** [comment]
I'm going to figure out how to put the "go-junit-report > result.xml" into code, so that I can see the output then get the result.xml

**Comment by jstemmer** [comment]
If you install the latest version (v2.0.0) then you can use the `-iocopy` flag to see the `go test` output, e.g.:
```
go test -v 2>&1 | go-junit-report -iocopy -out report.xml
```
